### DISCUSSION TITLE
I always get message that .Net Sdk 3.1 could not be located.

### DISCUSSION THREAD

**Discussion by jairofranchi** [microsoft/azuredatastudio#15586] [General]
I already tried to install and remove Azure Data Studio, clean all dotnet installed and then install again version 3.1, but I always get the same message when I will try to open a .sqlproj, I also tried to point to sdk path.

Is there a way that I can fix this problem? I'm using version 1.29.0 of Azure Data Studio.

![image](https://user-images.githubusercontent.com/744971/119741860-f0bb2f80-be5c-11eb-8bac-678db5faae31.png)

**Comment by jairofranchi** [comment]
This discussion can be closed, I get this working now.

**Comment by kisantia** [comment]
A new option was added to this notification to not show again https://github.com/microsoft/azuredatastudio/pull/15470 and will be in the next release
